ui/tests/integration/components/date-dropdown-test.js

```diff
@@ -1,23 +1,9 @@
-/* eslint qunit/no-conditional-assertions: "warn" */
+/* eslint-disable qunit/no-conditional-assertions */
```

Comment:
Curious if there is a way to refactor the test(s) in a way where there are not conditional assertions made? Possibly breaking them out into multiple tests, one for each condition.

Reply:
🤔 Hmm, the dropdown disables future months and years, depending on the month or year is selected. Either can be selected first, so which items are disabled in the list depends on each selection. 


I think a way I could refactor is pushing all the elements into their respective arrays (i..e `disabledMonths`) and then assert everything in that array is disabled...? But open to ideas

Reply:
I left a suggestion below for one of the specific cases where you could move the past year assertions to a different test and conditionally use `assert.false` or `assert.true`.